### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (74)
Screenshot (75)

Website Name : Sports Cave
This is the landing page for my E-commerce website.

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+.badge {
+    width: 1.25rem;
+    height: 1.25rem;
+    border-radius: 100%;
+    position: absolute;
+    text-align: center;
+    top: 0;
+    right: 0;
+    color: var(--bg-color);
+    font-size: 16px;
```
Comment (tonymike99) [inline css/navbar.css:26]: Can use rem here

**Review by tonymike99 — APPROVED** [review]
Hey! Nicely written code. And colourful UI.
Suggestion: I see that you have used px for values above 16. Change them to rem, it would be better. Plus, there is so much of code in just one PR. You can avoid doing that.
Otherwise, everything looks good!

File: css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.container {
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+}
```
Comment (abhijeetekad) [inline css/navbar.css:3]: You can use variable for box-shadow as well

File: css/product-wishlist.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    text-align: center;
+}
+.card-text{
+    text-align: center;
+    color: var(--accent-color);
+}
+.card-img{
+    width: 400px;
+    height: 300px;
+}
```
Comment (abhijeetekad) [inline css/product-wishlist.css]: Don't you px for bigger size.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://ozone-ui-library.netlify.app/styles.css">
+    <link rel="stylesheet" href="./css/navbar.css">
+    <link rel="stylesheet" href="./css/root.css">
+    <link rel="stylesheet" href="./style.css">
+    <title>Sports Cave</title>
```
Comment (abhijeetekad) [inline index.html:15]: You can put all your CSS files in one main file and use that file everywhere. That will be easy to maintain.

**Review by abhijeetekad — APPROVED** [review]

**Pull request merged**